common.mk

```diff
@@ -4,8 +4,11 @@ install-vendor: install-glide
 
 install-glide:
 		@which glide > /dev/null || (go get -u github.com/Masterminds/glide && cd $(GOPATH)/src/github.com/Masterminds/glide && git checkout v0.12.3 && go install)
-		@glide -version > /dev/null || echo "Glide install failed"
+		@glide -version > /dev/null || (echo "Glide install failed" && exit 1)
 
 install-ci:
 	make install-vendor
 
+install-metalinter:
+	@which gometalinter > /dev/null || (go get -u github.com/alecthomas/gometalinter && gometalinter --install)
```

Comment:
Need to also `go install github.com/alecthomas/gometalinter` before `gometalinter --install` otherwise Go won't build and put `gometalinter` in the binaries dir.